ls1intum/Artemis
# [Feature] [Orion] Download Submissions for Tutors

### Checklist

### Motivation and Context
Artemis's side of https://github.com/ls1intum/Orion/pull/56. See there for further details

### Description
- Adapts the existing code to the new 3 different ways of open exercises
- Extends the Orion connector with methods to call Orion's new functionality
- Presents buttons to Orion users to trigger the new functionality

### Steps for Testing
Has to be tested with https://github.com/ls1intum/Orion/pull/56, see there for testing steps

Note for code review: Several of the changes are due to running checkstyle, prettier and eslint and are not connected to the changes
